src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -669,6 +778,12 @@ static inline int MPIDI_NM_mpi_init_hook(int rank,
         mapped_table = (fi_addr_t *) av_attr.map_addr;
         for (i = 0; i < size; i++) {
             MPIDI_OFI_AV(&MPIDIU_get_av(0, i)).dest = mapped_table[i];
+#if (MPIDI_OFI_ENABLE_RUNTIME_CHECKS || MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS)
```

Comment:
@raffenet - The reason this needs to be a preprocessor check is because `ep_idx` won't be defined if one of these two macros isn't defined.

@yfguo - I think this version of the preprocessor check isn't portable. If I remember correctly, the Intel MPI team made me switch it to two different checks.